src/interface/web/app/components/navMenu/navMenu.tsx

```diff
@@ -140,220 +149,51 @@ export default function NavMenu() {
-                                    <div className="flex flex-rows">
-                                        <KhojSearchLogo className="w-6 h-6" />
-                                        <p className="ml-3 font-semibold">Search</p>
-                                    </div>
-                                </Link>
-                            </DropdownMenuItem>
-                        )}
-                        {userData && (
-                            <DropdownMenuItem>
-                                <Link href="/settings" className="no-underline w-full">
-                                    <div className="flex flex-rows">
-                                        <GearFine className="w-6 h-6" />
-                                        <p className="ml-3 font-semibold">Settings</p>
-                                    </div>
-                                </Link>
-                            </DropdownMenuItem>
-                        )}
-                        <>
-                            <DropdownMenuSeparator />
+                        {userData ? (
                             <DropdownMenuItem>
-                                <Link href="https://docs.khoj.dev" className="no-underline w-full">
+                                <Link href="/auth/logout" className="no-underline w-full">
                                     <div className="flex flex-rows">
-                                        <Question className="w-6 h-6" />
-                                        <p className="ml-3 font-semibold">Help</p>
+                                        <ArrowRight className="w-6 h-6" />
+                                        <p className="ml-3 font-semibold">Logout</p>
                                     </div>
                                 </Link>
                             </DropdownMenuItem>
+                        ) : (
                             <DropdownMenuItem>
-                                <Link
-                                    href="https://github.com/khoj-ai/khoj/releases"
-                                    className="no-underline w-full"
+                                <Button
+                                    variant={"ghost"}
+                                    onClick={() => setShowLoginPrompt(true)}
+                                    className="no-underline w-full text-left p-0 content-start justify-start items-start h-fit"
                                 >
-                                    <div className="flex flex-rows">
-                                        <Code className="w-6 h-6" />
-                                        <p className="ml-3 font-semibold">Releases</p>
+                                    <div className="flex flex-rows text-left content-start justify-start items-start p-0">
+                                        <ArrowRight className="w-6 h-6" />
+                                        <p className="ml-3 font-semibold">Login</p>
```

Comment:
Menu still shows `Logout` for logged out user instead of Login

Reply:
Addressed in #1015